tests/helics/apps/TracerTests.cpp

```diff
@@ -644,19 +644,20 @@ INSTANTIATE_TEST_SUITE_P(tracer_tests,
                          tracer_clone_file_tests,
                          ::testing::ValuesIn(simple_clone_test_files));
 
+/*
 #ifdef ENABLE_ZMQ_CORE
```

Comment:
The #ifdef looks like it should be updated if the test is no longer using ZMQ.

Reply:
I am playing with this test yet to see if it passes.  I will clean up once I figure what I want to do with it